### PR TITLE
opae: update OPAE spec file

### PR DESCRIPTION
**Pull request by anandaravuri**

### CODE REVIEW

File: tools/extra/pac_hssi_config/pac_hssi_config.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python3
```
Comment (tswhison) [inline tools/extra/pac_hssi_config/pac_hssi_config.py:1]: These scripts should go in a separate pull request.

Comment (anandaravuri) [reply tools/extra/pac_hssi_config/pac_hssi_config.py]: fixed , move to another pull request
